menu.kando.Kando.yml

```diff
@@ -0,0 +1,35 @@
+id: menu.kando.Kando
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: AppRun
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=x11
+  - --socket=wayland
+  - --talk-name=org.freedesktop.Flatpak
+  - --talk-name=org.gnome.Shell
+  - --talk-name=org.kde.KWin
```

Comment:
what are these for and how does it work when not on gnome or kde?

Reply:
Kando supports all X11-based desktop environments. For Wayland compositors it ships dedicated backends. The ones used on GNOME and KDE require the permissions above, on Hyprland the `hyperctl` tool and several wlroots-specific Wayland protocols are used. Other Wayland-only compositors are not supported ATM.